lib/options.js

```diff
@@ -104,16 +104,35 @@ exports.parse = function parse(argv) {
     else if(option === '--no-profile') {
       options.profile = false;
     }
-    else if(option === '--no-channel') {
-      options.channel = false;
-    }
-    else if({'--cluster':1,'--port':1,'--addr':1,'--path':1}[option]) {
-      i++; // Consume the option's argument
+    else if(option === '--cluster') {
+      i++;
     }
-    else if(/^--(cluster|port|addr|path)=/.test(option)) {
+    else if(/^--cluster=/.test(option)) {
     }
     else if(option === '--no-cluster') {
     }
+    // Only -C,--control,--no-control are documented, now, to align with
+    // strong-pm. The others are legacy, to be deleted sometime.
+    else if({
+      '-C':1, '--control':1, '--port':1, '--addr':1, '--path':1
```

Comment:
Should these be `true`'s instead of `1`'s? This almost looks like it is defaulting to listening on port 1 or a socket at `./1`.